### PR TITLE
add product multi select

### PR DESCRIPTION
**Pull request by lucas-larsson**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by lucas-larsson** [comment]
* **#4759** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4759?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lucas-larsson and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/app/debugger/seb-leads/createSebLead.tsx [hunk 1]
```diff
@@ -1,108 +1,124 @@
...
+            } else {
+                products = products + ',condoInsurance'
+            }
+        } else if (products.includes('condoInsuranceBrf')) {
+            maybeProductSubType = 'condoInsurance.condoInsuranceCondominium'
+            products = products.replace('condoInsuranceBrf', 'condoInsurance')
+        } else if (products.includes('condoInsuranceRent')) {
+            maybeProductSubType = 'condoInsurance.condoInsuranceRental'
+            products = products.replace('condoInsuranceRent', 'condoInsurance')
+        }
```
Comment (lucas-larsson) [inline apps/store/src/app/debugger/seb-leads/createSebLead.tsx:38]: A whole bunch of `if/else` statements to handle how the request should look like for the SEB API, I will try to refactor it to be nicer, but just committed this for the demo

File: apps/store/src/app/debugger/seb-leads/createSebLead.tsx [hunk 2]
```diff
@@ -1,108 +1,124 @@
...
-  }
-  let sebSessionId: string | null
-
-  try {
-    const result = await createSebLeadStaging(parameters)
-    sebSessionId = result.sessionId
-    if (!sebSessionId) {
-      throw new Error(`No sessionId in response (${JSON.stringify(result)})`)
+    // very ugly code to handle the way SEB wants the productType and productSubType
+    if (products !== null && typeof products === "string") {
```
Comment (alebedev) [inline apps/store/src/app/debugger/seb-leads/createSebLead.tsx:21]: This is not very easy to read in current form
- transforming `products` from internal format to whatever Insurely expects is one level of abstraction lower than other code here
- doing it as string manipulation sequence makes it harder

Suggestions
- extract function for conversion
- do it on array and convert to string at very last step

**Review by alebedev — COMMENTED** [review]
Please format everything with prettier, there's lots of formatting changes adding the noise otherwise

File: apps/store/src/app/debugger/seb-leads/createSebLead.tsx (hunk 2 above)
Comment (lucas-larsson) [reply apps/store/src/app/debugger/seb-leads/createSebLead.tsx]: Fixed, thanks for the suggestions

**Comment by lucas-larsson** [comment]
Did it know, had another prettier config that effected this

File: apps/store/src/app/debugger/seb-leads/CreateSebLeadsForm.tsx [hunk 3]
```diff
@@ -18,19 +23,30 @@ const PRODUCT_OPTIONS = [
...
+  const [selectedProducts, setSelectedProducts] = useState<Array<{ name: string; value: string }>>(
+    [],
+  )
+
+  const handleDelete = (value: string) => () => {
+    const newSelectedProducts = selectedProducts.filter((product) => product.value !== value)
+    setSelectedProducts(newSelectedProducts)
+  }
+
+  const items = PRODUCT_OPTIONS.filter((product) =>
```
Comment (alebedev) [inline apps/store/src/app/debugger/seb-leads/CreateSebLeadsForm.tsx:38]: `unselectedItems`?

File: apps/store/src/app/debugger/seb-leads/CreateSebLeadsForm.tsx [hunk 4]
```diff
@@ -18,19 +23,30 @@ const PRODUCT_OPTIONS = [
...
+    selectedProducts.every((selectedProduct) => selectedProduct.value !== product.value),
+  )
+
+  const [state, formAction] = useFormState(createSebLead, null)
+
+  const handleAdded = (product: any) => {
+    console.log('product:', product)
```
Comment (alebedev) [inline apps/store/src/app/debugger/seb-leads/CreateSebLeadsForm.tsx:45]: ?

**Review by alebedev — APPROVED** [review]

**Comment by lucas-larsson** [comment]
### Merge activity

* **Sep 24, 2:29 AM EDT**: @lucas-larsson started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4759).

**Pull request merged**